coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -1704,6 +1784,948 @@
+                "cid": "dsfis23sd"
+                "value": "HELPFUL" | "NOT_HELPFUL",
+                "rid": "bnm565j",
+                "mid": "sdddfe",
+                "reset": True # to undo any rating
+            }
+        '''
+        logger.info("Saving or Removing: User Resource")
+        tx = self.driver.session()
+        
+        reset_status = False
+        if rating.get("reset") != None and rating["reset"] == True:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)